src/fhir/input/fsh/profiles/HealthcareServiceDirectory.fsh

```diff
@@ -14,8 +14,12 @@ organisations specific for german Healthcare and Telematics Infrastructure.
 * meta.tag[Origin].system = "https://gematik.de/fhir/directory/CodeSystem/Origin"
 * id ..1 MS
```

Comment:
Wir wollen in einem HeathcareService auch eine TelematikID eintragen können (optional).
Manche Organisationen haben sich mehrere SMC-B Karten mit verschiedenen TelematikIDs geben lassen. Wenn diese Karten im Sinne eines HeathcareService genutzt werden, dann wollen wir diese TelematikID auch hier ablegen.
Zum Beispiel
* identifier 0..* MS
* identifier ^slicing.discriminator.type = #value
* identifier ^slicing.discriminator.path = "system"
* identifier ^slicing.rules = #open
* identifier contains TelematikID 0..1 MS
* identifier[TelematikID] only $IdentifierTelematikID

Reply:
@gem-uhe Wenn ich dich richtig verstehe, schlägst du hier eine Änderung vor, um mehrere TelematikIDs für eine Helathcare Organisation vornehmen zu können. Dem spricht aus meiner Sicht nix entgegen.
@patrick-werner Was meinst du?

Reply:
@SvenSommer Bisher können sie Organisationen beliebig viele SMC-Bs ausstellen lassen. Dabei können sie sich wünschen, ob die zusätzlichen SMC-Bs die gleiche TelematikID wie die Hauptkarte oder andere TelematiIDs haben. Jede SMC-B TelematikID wird in den LDAP VZD als Organisation eingetragen. Mit der Synchronisierung dieser LDAP Organisationseinträge kommen jetzt die Überlegungen, wie man sie sinnvoll auf eine Organisationsstruktur abbildet. Was macht man mit solchen Einträge im LDAP VZD
- DocMorris
- DocMorris KIM
oder
- MVZ am Brüderkrankenhaus Montabaur
- MVZ am Brüderkrankenhaus Montabaur
Deshalb war die Idee:  Wenn es sich bei den LDAP Einträgen um eine Unterabteilung/Service einer Organisation handelt, generieren wir dafür in FHIR einen HealtcareService und legen die TelematikID der SMC-B in diesem HealtcareService ab. Analog könnte der Typ der Organisation von solchen Einträgen aus dem LDAP VZD auf den FHIR HealthcareService.type abgebildet werden. Das passt wahrscheinlich nicht immer. Wir hatten bisher aber keine bessere Idee.

Reply:
Einfacher UseCase: 
eine Organization hat mehrere TelematikIDs -> identifier Kardinalität -> 1..*
--> mehrere slices, ein slice HAuptTelemeatikID, der andere slice die alternativen TelematikIds (per type)

Komplexer UseCase: Mehrere Standorte eines Klinikverbunds
part-of & inactive:
Falls eine Organization auf inactive gesetzt wird, müssen alle Kinder (revinclude per part-of) auch auf inactive gesetzt werden.

Wird nochmals innerhalb Gefyra/Gematik reflektiert.

Reply:
partOf & multiple TelematikID-Slices umgesetzt